### ISSUE TITLE
First layer width doesn't adjusted based on first layer height

### ISSUE THREAD

**Issue by erictioh** [supermerill/SuperSlicer#2186]
### What happened?

First layer width is currently intact despite i adjust the first layer height. When i adjust first layer height higher and the first layer width remain the same, it causes issue where it cannot stick well on bed because width is not enough.

### Project file & How to reproduce

see the what happen

![image](https://user-images.githubusercontent.com/44055730/148333313-dcea7bec-f79a-4097-8a43-9849880b8000.png)

### Version

2.3.57.9

### Operating system

Windows 10

### Printer model

Creality CR10 V3

**Comment by neophyl** [comment]
Not sure I'm understanding the issue.  Those settings are independent in Super Slicer.  In Prusa Slicer they are linked but not with SuSi.  If you want your first layer extrusion width to be wider then just set it to the width you want.

**Comment by supermerill** [comment]
Also, they removed the link in PS 2.4.

Which tooltip makes you think the width is dependant on the height? I have to fix it.

**Comment by erictioh** [comment]
Ok, you can see that the tool tip for first layer stated that this is influenced by default layer height. And i did a test indeed that if i increase the default layer height this first layer width is increased automatically.
![image](https://user-images.githubusercontent.com/44055730/148385715-27b0bbba-ba39-4132-a9e8-7e5715ced49a.png)

Then very intuitively i will ask myself, it should be that first layer width influenced by first layer height isn't it ? that's what lead me to open this ticket.

**Comment by neophyl** [comment]
Ah I see.  There's the relationship between width and spacing.  Changing one changes the other in the settings fields.  The exact relationship though does take into account the layer height as well as the overlap percentage.  So its not a direct change, its changing the math between the 2.
So for example a first layer width of 150% at a layer height of 0.1 produces a spacing of 145%, increase the layer height to 0.2 and the same width of 150% produces a spacing of 139%

Not sure what wording would make that better though.

**Comment by erictioh** [comment]
Yes check the screenshot of the 2 value of layer height which does change the first layer spacing or first layer width (depends on what i update either at the width or spacing before hand).

![image](https://user-images.githubusercontent.com/44055730/148399202-924ad38c-ce66-4c40-aa37-54f8869969e6.png)
![image](https://user-images.githubusercontent.com/44055730/148399321-227ed421-f837-4ed8-92c0-f53293cf7deb.png)

however my question is since layer height does change width/spacing, why not first layer height change the first layer width and spacing ?

**Comment by erictioh** [comment]
layer height do change almost all the param in the Extrusion width, however first layer height doesn't change a bit of any width, the impact is the higher first layer heigh is introduced, i have to manually guessing calculate the first layer width. However this is not the case for the rest of layer where the slicer calculate that out automatically for me. 

Not sure the way i see is correct or perhaps something hidden that i don't aware, you guys can further comment

**Comment by neophyl** [comment]
I'm not sure I understand your question so I'll leave that to Merill.

**Comment by supermerill** [comment]
that's because you set the first layer width and not the spacing. SO the width is the field 'set' and the spacing is the field 'computed from width and layer height'
the other fields are the opposite: spacing is set and width is computed.
You can look at the colour of the label to see who's set (light grey means computed)
You can have the meaning of the colour (and change it) when clicking on the '?'
